code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -30,6 +30,7 @@
 		handle_aura_emiter()
 
 	handle_aura_receiver()
+	handle_environment()
```

Comment:
This shouldnt be on Life() because it causes extra processing

Reply:
done